app/controllers/api/v1/rent_controller.rb

```diff
@@ -31,6 +39,37 @@ def rent_response
       def rent_serializer
         Api::V1::Serializer::RentSerializer
       end
+
+      def ranking
+        ranking = []
+        Book.all.each do |book|
```

Comment:
Please review the SQL statements, `order by` in SQL 
https://guides.rubyonrails.org/active_record_querying.html#retrieving-objects-from-the-database